config/mail.php

```diff
@@ -84,8 +84,8 @@
     */
 
     'from' => [
-        'address' => env('MAIL_FROM_ADDRESS', 'hello@example.com'),
-        'name' => env('MAIL_FROM_NAME', 'Example'),
+        'address' => env('MAIL_FROM_ADDRESS', 'PHPOE29@gmail.com'),
+        'name' => env('MAIL_FROM_NAME', 'Nguoi_La'),
```

Comment:
config xàm xí vậy à? (luom)
```suggestion
        'name' => env('MAIL_FROM_NAME', 'Ten web'),
```